src/Context/Authentication/LoginContext.js

```diff
@@ -0,0 +1,58 @@
+import axios from "axios";
+import React from "react";
+import { useState } from "react";
+import { createContext } from "react";
+import { useContext} from "react";
```

Comment:
import it like this: 

` import React, { useState, createContext, useContext } from "react"; `